.env.test

```diff
@@ -1,12 +1,12 @@
 # Flask server configuration.
-FLASK_APP=app
+FLASK_APP=todo_app/app

```

Comment:
Minor, but it would be nice to also apply this in the `.env.template` file so that new developers also have this set correctly

Reply:
I have amended as advised